src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import MainPage from './components/MainPage/MainPage.jsx';
```

Comment:
could you read about absolute/relative path and implement it here, please?
so instead of having `./components/MainPage/MainPage.jsx'` we should have `components/MainPage/MainPage`
also regards the `.jsx` extension - are you sure that you need it there?